ajax/libs/elfinder/package.json

```diff
@@ -24,7 +24,7 @@
       {
         "basePath": "",
         "files": [
-          "js/**/+(elFinder|elfinder|quicklook.googledocs)*.+(js|map)",
+          "js/**/*.+(html|js|map)",
```

Comment:
@nao-pon Thank you.
May I ask that in this rule, CDNJS will get the files such as `js/commands/*` and `js/i18n/help/jp.html` but how can web front-end developers use with HTML tag like `<script src="js/i18n/help/jp.html"></script>`?

If you think these files not useful to front-end developer, would you change as the followings?
`"js/**/*+(elFinder|elfinder|quicklook.googledocs)*.+(js|map)"`
Or if the `js/commands/*` is still useful, let's change to the followings:
`js/**/!(*encoding-japanese.min)*.+(js|map)`

Note: 
CDNJS need to add this library: https://github.com/polygonplanet/encoding.js (js/extras/encoding-japanese.min.js)

Thank you.